README.md

```diff
@@ -1,3 +1,53 @@
+- Getting started with ship-secure-frontend project
+
+# Cloning
+
+From https://github.com/Snowwhite-Workshop/ship-secure-frontend copy the url under code section
+
+Go to your command prompt or terminal, navigate to the folder you wish to work on and type the following command
+
+$ git clone THE_URL_YOU_COPIED_FROM_GITHUB (ensure that your internet is up). Then,
+
+$ cd ship-secure-frontend/ (the newly cloned folder)
+
+# Install project dependencies
+
+$ npm install (or yarn install - if you are using yarn)
+
+Then,
+$ npm start (to start your application)
+
+# Creating your own local branch
+
+Type the following commands in your terminal
+
+$ git checkout -b NEW_BRANCH_NAME (to create and switch to your new branch)
+
+// make your changes in your branch$ //
+
+# Committing and Pushing to github
+
+$ git add . (to stage all your changes for commit)
+
+$ git commit -m "YOUR_COMMIT_MESSAGE"
+
+Then,
+
+$ git push origin NAME_OF_YOUR_BRANCH
+
+$
+
+# Creating a pull request
+
+// Goto https://github.com/Snowwhite-Workshop/ship-secure-frontend to immediatly see your commits in your branch.
+// You would be promted to create a pull request or you may navigate to "pull requests" tab and see create a pull request for your branch.
+
+- select the appropriate branch you wish your own branch to be merged into,
+- add comments where necessary
```

Comment:
comments are actually compulsory. so that we know what the pull request is about